man/zypper-docker-up.1.md

```diff
@@ -9,9 +9,11 @@ zypper\-docker update \- Install the available updates for the given image.
 
 # DESCRIPTION
 The **update** command updates the given openSUSE/SUSE Linux Enterprise image
-with all the available updates. The updated image will have a new name, as
-provided by the NEW-IMAGE argument. Note that both the IMAGE and the NEW\-IMAGE
-arguments have to follow Docker's naming format.
+with all the available updates. If there is a zypper-related update, which returns
+the exit code 103 (zypperExitInfRestartNeeded), zypper-docker goes through the 
```

Comment:
The actual exit code of zypper is called `ZYPPER_EXIT_INF_RESTART_NEEDED` (see zypper's manpages). After that change, I think we're good to go :+1:

Reply:
ok, will change that